frontend/src/pages/clusterSettings/ClusterSettings.scss

```diff
@@ -0,0 +1,14 @@
+.odh-cluster-settings {
+  .pf-c-form__group {
+    border: 1px solid var(--pf-global--BorderColor--100);
+    padding: 15px;
+    margin: 0px
+  }
+  .odh-number-input{
+    max-width: 40px;
+  }
+  .pf-c-input-group {
+    padding-top: 10px;
+    padding-bottom: 10px;
```

Comment:
PF Vars?

Reply:
removed this it's not needed for this PR.